jose-jwt/JWTSettings.cs

```diff
@@ -28,7 +28,7 @@ public class JwtSettings
                 { JwsAlgorithm.ES256, new EcdsaUsingSha(256) },
                 { JwsAlgorithm.ES384, new EcdsaUsingSha(384) },
                 { JwsAlgorithm.ES512, new EcdsaUsingSha(521) }
-#elif NETSTANDARD1_4 || NET461

```

Comment:
Why removing NETSTANDARD1_4 ?

Reply:
Not 'intentional'. However it wasn't a target I was going to be testing / using.
I could re-work this to be less / no code change for other targets - I didn't know if you would plan on using the PR to actually merge in - or if you would make a similar functional change 'from scratch'.
Let me know if you want me to rework this PR to be less impacting existing targets?

Reply:
Overall looks good to be used as basis for update. Can we:

  - make PR to new branch `netcore2.1` instead of `master`
  - join #if/#elif with or (||) (NETSTANDARD1_4 || NETSTANDARD2_1) where applicable?

Reply:
Also i think, is there are any issues for using lib targeted at NETSTANDARD1_4 on NETSTANDARD2_1 (except obvious new stuff in one place)? Probably we don't even need that joined compilation condition?